### PR TITLE
Add commonmark as a dependency

### PR DESCRIPTION
**Pull request by dpryan79**

### CODE REVIEW

**Comment by dpryan79** [comment]
[grumble] no python36 commonmark package [/grumble]

**Comment by dpryan79** [comment]
Once https://github.com/conda-forge/commonmark-feedstock/pull/2 is merged this should start working.

**Comment by dpryan79** [comment]
This should then fix #5

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,17 +13,19 @@ source:
...
-  number: 0
+  number: 1
   script: python setup.py install --single-version-externally-managed --record=record.txt
 
 requirements:
   build:
     - python
     - setuptools
+    - commonmark >=0.5.6
     - sphinx
```
Comment (jakirkham) [inline recipe/meta.yaml]: So we have been moving to keep only what is strictly required in `build`. In this case, probably just `python` and `setuptools`.

Comment (dpryan79) [reply recipe/meta.yaml]: Thanks @jakirkham, that makes sense. I've updated the recipe.

**Review by croth1 — APPROVED** [review]

**Comment by dpryan79** [comment]
@croth1 Could you merge this when you get a chance?

**Pull request merged**